app/components/DocumentBreadcrumb.tsx

```diff
@@ -96,15 +98,20 @@ function DocumentBreadcrumb(
   const items = React.useMemo(() => {
     const output = [];
 
-    if (category) {
-      output.push(category);
+    if (!reverse) {
```

Comment:
Repeating this logic twice is not good, it would be simpler to reverse the breadcrumb at the end after all other logic is complete